validator/client/validator_attest_test.go

```diff
@@ -15,6 +15,10 @@ import (
 	logTest "github.com/sirupsen/logrus/hooks/test"
 )
 
+func init() {
+	delay = 0
```

Comment:
I would like to see more tests here. 

You could do something like this:
- Set delay to 25ms
- Call the method
- See that the attestation has not been broadcasted
- Sleep for 30ms
- See that the attestation has been broadcasted now, after the delay